### ISSUE TITLE
Change History Margin - reloading document

### ISSUE THREAD

**Issue by Umbracinis** [notepad-plus-plus/notepad-plus-plus#12319]
First time posting in GitHub, so apologies if I am doing something wrong.

I have just updated to the latest version of Notepad++ (v8.4.6 - 64bit) on my Windows 11 Pro.
I really like the change history margin. I did however find a slight annoyance with it:

If I reload the document (Reload from disk, or CTRL + R) then the history margin of the entire document is marked with a green bar. This then makes it more difficult to distinguish new changes, as they get 'lost' in the green bar, defeating a bit its purpose. The only way I could find to 'clear' the history margin again was to close the document and then open it again.

I believe a better option would be to simply 'clear' the history margin on reload, as if the document had just been opened for the first time.

Thank you for your time.

**Comment by alankilborn** [comment]
>First time posting in GitHub

Welcome to the Notepad++ project!

>so apologies if I am doing something wrong

Well, you should have followed the issue creation template, but... your statements are clear enough and you have the version of N++ you are running in there... so it is OK.

Also, before creating a new issue, you should examine existing open issues to see if yours is already reported, and maybe even recently closed ones (to see if it is already fixed).  In this case, it has been reported, see #12261 

The bottom line is that _Change History_ is a really new feature of Notepad++'s editing component ([Scintilla](https://www.scintilla.org/)), and as such, will have a few "growing pains" such as this one.  I'm sure it will be corrected, but as Scintilla is a completely separate project than Notepad++, it may take a bit of time for a fix.

Recommend CLOSURE of this item as duplicate.

**Comment by Umbracinis** [comment]
Firstly, thank you very much for your quick response.

I did look at the creation template, but it seemed a bit overkill for this, so I decided to clearly state my issue in a more succinct manner while still providing enough tech info. I shall follow the template next time, no matter what.

I shall now close this and follow the one you mentioned.

Again, apologies for the mistakes, and thank you for your tolerance and useful response.

CLOSED as duplicate. See https://github.com/notepad-plus-plus/notepad-plus-plus/issues/12261
